Lecture2/task1/Vegetable.java

```diff
@@ -0,0 +1,12 @@
+abstract public class Vegetable {
```

Comment:
first we should set access modifier (public,private..etc)
public abstract class ...

Reply:
fixed